### ISSUE TITLE
comfortably native completions

### ISSUE THREAD

**Issue by ilyazuev** [SublimeHaskell/SublimeHaskell#349]
I have a suggestion.
You can mix (hsdev) autocompletion with Sublime's native autocompletion.
I did it already in my local copy
By my opinion it is very comfortably

```
#SublimeHaskell\autocomplete.py

# add new function to collect native completions
def make_sublime_completions(prefix, point):
    return list(map(
            lambda k: (k + '\tin file', k),
            sublime.active_window().active_view().extract_completions(prefix, point)))
			# ... or "extract_completions" for all views in "active_window"
...	
	@hsdev.use_hsdev([])
	def get_completions(self, view, locations):
	...
		else:
			with self.cache as cache_:
				...
				# line 250: first native completions in active file(s), then completions from plugin
				return make_sublime_completions(qualified_prefix, locations[0]) + self.keyword_completions + completions
```

**Comment by ilyazuev** [comment]
... or from many files in the window

```
return list(sum(list(map(
				lambda v: (list(map(lambda k: (k + '\t' + v.file_name().split('\\')[-1], k), v.extract_completions(prefix, point)))),
				sublime.active_window().views())), []))
```

**Comment by bscottm** [comment]
@ilyazuev: Can you update/upgrade to the latest SublimeHaskell (2.0.3) and see if completions work better for you?

**Comment by bscottm** [comment]
@ilyazuev: This looks like a duplicate of issue #366. Closing this issue -- feel free to re-open if the fix in issue #366 isn't correct.
